recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "roaring-landmask" %}
-{% set version = "0.7.1" %}
+{% set version = "0.9.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/roaring_landmask-{{ version }}.tar.gz
-  sha256: 0a289eaf276f59f46f7924faa5cf058b3b6d2be02a0f437c1466b572f75f0aef
+  sha256: dddd74951ca1b01c624ac2973def602dad6c69cf5f40e9bd64bfac84bc820889
 
 build:
-  number: 3
+  number: 1
+  skip: true  # [win and vc<14]
```

Comment:
This skip shouldn't be necessary, since we always use vc>=14 on conda-forge at this point.

Reply:
Ok, I got it from here: https://github.com/conda-forge/geos-feedstock/blob/main/recipe/meta.yaml#L15

Reply:
Yeah, we should remove that there, too.

Reply:
You can see that vc is pinned to 14 for conda-forge builds here (and has been for 4 years now):
https://github.com/conda-forge/conda-forge-pinning-feedstock/blame/main/recipe/conda_build_config.yaml#L898
So there shouldn't be any feedstocks at this point that use vc<14, I think.  Somehow, the skip never got removed on the geos-feedstock, presumably because it does no harm.

Reply:
Ok. Hm.. Then this should not make a difference here either. Somehow it does not link to c++ stdlib on windows.

Reply:
Maybe it is related to: https://github.com/conda-forge/geos-feedstock/blob/main/recipe/patches/0001-unconditional-geos-pc.patch (@magnusuMET any idea?)

Reply:
Skipping the windows build for now.